examples/Accounts API/README.md

```diff
@@ -0,0 +1,30 @@
+# Duo Accounts API Examples Overview
+
+
+## Examples
+
+This folder contains various examples to illustrate the usage of the `Accounts` module within the 
+`duo_client_python` library.  The Duo Accounts API is primarily intended for use by Managed Service 
+Partners (MSP) to assist in the automation of managing their child (customer) Duo accounts.
+
+Use of the Duo Accounts API requires special access to be enabled. Please see the 
+[online documentation](https://www.duosecurity.com/docs/accountsapi) for more information.
+
+# Using
+
+To run an example query, execute a command like the following from the repo root:
+```python
+$ python3 examples/Auth/get_billing_and_telephony_credits.py
```

Comment:
AFAICT the path is actually examples/Accounts API/get_billing...?

Should we dispense with the spaces in the directory name though?  path spaces can be problematic